### PR TITLE
Move tests to relevant files

### PR DESCRIPTION
**Pull request by sonalk0209**
# Problem Statement
Created 2 files, one for data+server pod restart and one for multi data pod restart. Move relevant tests from single data pod restart to these files.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/ha/single_pod_restart/test_data_pod_restart.py [hunk 1]
```diff
@@ -82,8 +79,7 @@ def setup_class(cls):
         cls.random_time = cls.s3_clean = cls.test_prefix = cls.test_prefix_deg = None
```
[the remark this answers is not in the document]
Comment (sonalk0209) [reply tests/ha/single_pod_restart/test_data_pod_restart.py:78]: Removed host_worker_list from all 3 files. Rest are needed so kept them.

**Review by ankita-93 — APPROVED** [review]

File: tests/ha/single_pod_restart/test_data_server_pod_restart.py [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    Test suite for single Data and Server Pod Restart
+    """
+
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup operations for the test file.
+        """
+        LOGGER.info("STARTED: Setup Module operations.")
+        cls.csm_user = CMN_CFG["csm"]["csm_admin_user"]["username"]
```
Comment (rahul-hatwar) [inline tests/ha/single_pod_restart/test_data_server_pod_restart.py:58]: we can remove this, not using anywhere

File: tests/ha/single_pod_restart/test_data_server_pod_restart.py [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+        cls.csm_passwd = CMN_CFG["csm"]["csm_admin_user"]["password"]
```
Comment (rahul-hatwar) [inline tests/ha/single_pod_restart/test_data_server_pod_restart.py:59]: we can remove this

File: tests/ha/multi_pod_restart/test_multi_data_pod_restart.py [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+    Test suite for Multiple Data Pod Restart
+    """
+
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup operations for the test file.
+        """
+        LOGGER.info("STARTED: Setup Module operations.")
+        cls.csm_user = CMN_CFG["csm"]["csm_admin_user"]["username"]
```
Comment (rahul-hatwar) [inline tests/ha/multi_pod_restart/test_multi_data_pod_restart.py:58]: remove this, we are not using

File: tests/ha/multi_pod_restart/test_multi_data_pod_restart.py [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+        cls.csm_passwd = CMN_CFG["csm"]["csm_admin_user"]["password"]
```
Comment (rahul-hatwar) [inline tests/ha/multi_pod_restart/test_multi_data_pod_restart.py:59]: same, remove this

File: tests/ha/multi_pod_restart/test_multi_data_pod_restart.py [hunk 6]
```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/python  # pylint: disable=R0902
```
Comment (rahul-hatwar) [inline tests/ha/multi_pod_restart/test_multi_data_pod_restart.py:1]: add this pylint disable in words, to sync the code between all files

**Review by ApurwaMohite — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Review by rahul-hatwar — APPROVED** [review]

**Comment by sonalk0209** [comment]
image

**Pull request merged**